noir-projects/noir-contracts/contracts/token_contract/src/main.nr

```diff
@@ -21,12 +21,10 @@ pub contract Token {
     use dep::aztec::{
         context::{PrivateCallInterface, PrivateContext},
         encrypted_logs::log_assembly_strategies::default_aes128::{
-            // TODO(benesjan): uncomment this once the events are updated.
-            // event::encode_and_encrypt_event_unconstrained,
+            event::encode_and_encrypt_event_unconstrained,
             note::{encode_and_encrypt_note, encode_and_encrypt_note_unconstrained},
         },
-        // TODO(benesjan): uncomment this once the events are updated.
-        // event::event_interface::EventInterface,
+        event::event_interface::EventInterface,
```

Comment:
Events are now processed in Noir so we no longer need to do this to avoid the error of the event logs being passed to process_log.